packages/components/pie-toggle-switch/src/defs.ts

```diff
@@ -1,6 +1,15 @@
 import { RTLComponentProps } from '@justeattakeaway/pie-webc-core';
 
+export type AriaProps = {
+    label?: string,
+    describedBy?: string
+};
+
 export interface ToggleSwitchProps extends RTLComponentProps {
+    /**
+     * The ARIA labels used for the mtoggle switch.
```

Comment:
think `mtoggle` might be a typo

Reply:
i believe that 'toggle switch' might have been 'modal' before the copy paste 😬